pages/biosero_cherrypick.vue

```diff
@@ -76,17 +76,28 @@ export default {
       const response = await lighthouseBiosero.createDestinationPlateBiosero(form)
       this.handleResponse(response)
     },
+    handleResponse(response) {
+      let message, type
+      if (response.success) {
+        message = response.response
+        type = 'success'
+      } else {
+        message = response.error.message
+        type = 'danger'
+      }
+      this.showAlert(message, type)
+    },
     async fail(form) {
       const response = await lighthouseBiosero.failDestinationPlateBiosero(form)
-      this.handleResponse(response)
```

Comment:
handleResponse and handleFailResponse are very similar. To fix this you could change the response from lighthouseBiosero.failDestinationPlateBiosero to return the same data structure. Think this can be done by returning the whole error object in the response